<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMasterTestDlg</class>
 <widget class="QDialog" name="AddMasterTestDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>488</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Select test</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblMasterTest">
         <property name="text">
          <string>Master test:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSelectedTest">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&lt;not selected&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="trvTests">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lblCondition">
         <property name="text">
          <string>Condition:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbCondition">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Perform test if master test is &quot;Alive&quot;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Perform test if master test is &quot;Alive&quot; or &quot;Unknown&quot;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Perform test if master test is &quot;Dead&quot;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Perform test if master test is &quot;Dead&quot; or &quot;Unknown&quot;</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddMasterTestDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddMasterTestDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
